pandas/core/dtypes/generic.py

```diff
@@ -139,7 +139,13 @@ def _subclasscheck(cls, inst) -> bool:
         "ABCExtensionArray",
         "_typ",
         # Note: IntervalArray and SparseArray are included bc they have _typ="extension"
-        {"extension", "categorical", "periodarray", "datetimearray", "timedeltaarray"},
+        {
+            "extension",
```

Comment:
Am I correct in seeing that just for formatting here changed? i.e. There is no new elements added to this set?

Reply:
## short story

Indeed the set content have not changed.

## long story

I first tried to resolve the issue adding one element to the set which induces black to change formatting. I realized afterwards that ABCNumpyExtensionArray already existed so I finaly used it. I forgot at the end to revert the formating change for ABCExtensionArray

Reply:
Thanks, could you undo the formatting here?